src/vega.schema.ts

```diff
@@ -972,6 +972,8 @@ export type Cursor = 'auto' | 'default' | 'none' |
   'row-resize' | 'all-scroll' | 'zoom-in' |
   'zoom-out' | 'grab' | 'grabbing';
 export type StrokeCap = 'butt' | 'round' | 'square';
+export type StrokeJoin = 'miter' | 'round' | 'bevel';
```

Comment:
Shouldn't we use typings from Vega typing?

Reply:
We don't have it over there yet. Vega-typings just use string for now so we need to fix that first.